modules/admin_manual/pages/configuration/server/security_setup_warnings.adoc

```diff
@@ -41,6 +41,16 @@ Transactional file locking is disabled, this might lead to issues with race cond
 Please see xref:configuration/files/files_locking_transactional.adoc[Transactional File Locking] 
 for how to correctly configure your environment for transactional file locking.
 
+== Background Jobs
+
+```
+We recommend to enable system cron as any other cron method has possible performance and reliability implications.
+```
+
+We recommend using cron as the background job scheduler.
+
+Further Information can be found in the docs article on https://doc.owncloud.com/server/admin_manual/configuration/server/background_jobs_configuration.html[Background Jobs]
```

Comment:
You NEVER do a reference to a part of the ownCloud doumentation using http(s)...
You MUST use xref:...
See the best practices guide for how to do that